pages/api/groups/[type]/[slug]/users/[userId].ts

```diff
@@ -40,21 +41,22 @@ const loadGroupItemMiddleware = async (req, res, cb) => {
     })
 }
 
-const getUsers = async (req, res) => {
-  res.status(200).json(req.item.members)
-}
-
 const addMember = async (req, res) => {
+  const {
+    query: {
+      type: { slug },
+      body,
+      item: { id },
+      currentUser,
+    },
+  } = req
 
-  updateOneContent(type.slug, req.item.id, {
-    members: [
-      ...req.item.members,
-      req.body
-    ]
+  updateOneContent(slug, id, {
+    members: [...req.item.members, body],
```

Comment:
i think you need to add also the uniqBy here.

This endpoint is in development , right?

Reply:
correct